migration-engine/core/tests/existing_data_tests.rs

```diff
@@ -70,8 +70,10 @@ fn adding_a_required_field_must_use_the_default_value_for_migrations() {
                 myfloat Float @default(2)
                 boolean Boolean @default(true)
                 string String @default("test_string")
-                dateTime DateTime 
-                enum MyEnum @default(C)
+                dateTime DateTime
+                // TODO: Currently failing because of ambiguity concerning expressions. Pending on
+                // spec work.
+                // enum MyEnum @default(C)
```

Comment:
What's the problem here?

Reply:
There's a validation error from this line: https://github.com/prisma/prisma-engine/blob/188a379c9b8c6650e5812f9bdb7407d7b197692f/libs/datamodel/src/validator/lift.rs#L131

It would be easy to make it work, since it feels like it should work, but it felt out of scope for this PR.